apps/src/aichat/types.ts

```diff
@@ -13,6 +13,7 @@ export enum Role {
   ASSISTANT = 'assistant',
   USER = 'user',
   SYSTEM = 'system',
+  MODEL_UPDATE = 'update',
```

Comment:
I think we'll need to filter out this new role / message type in what we send to our back end...LMK if anyone has alternative ideas on where/how we should store these model update messages.

Reply:
Yeah, I think when we invoke the chat API we'll just filter out messages of this type.